.github/workflows/performance.yml

```diff
@@ -2,13 +2,8 @@
 name: Performance Regression Testing
 # Schedule triggers
 on:
-  # TODO this is just while developing
-  pull_request:
-    branches:
-      - 'develop' 
-      - 'performance-regression-testing'
+  # runs twice a day at 10:05am and 10:05pm
   schedule:
-    # runs twice a day at 10:05am and 10:05pm
```

Comment:
I don't mind leaving this comment since its easy to see when to expect the runs

Reply:
it's there! I just moved it.